CONTRIBUTING.md

```diff
@@ -128,3 +129,23 @@ You may also check what is the last `index-state` available within the current h
 ```console
 $ nix eval .#last-index-state
 ```
+
+## Roles and Responsibilities
+
+The `cardano-prelude` package is collectively mainntained by the `cardano-core`
+team.  If you open a PR expect a review from one of the core contributors.
+
+## CI
+
+`cardano-prelude` is build with GitHub Actions (using `cabal`) and cicero
```

Comment:
s/build/built

Reply:
Actually this part should be removed, there already is a section about `CI`.